### DISCUSSION TITLE
Rich Text formatting - blockquotes etc

### DISCUSSION THREAD

**Discussion by jumptrnr** [tinacms/tinacms#3804] [Help] [answered]
Hi all - got the nextjs app going - very basic formatting shows up - like bold and images, but blockquotes for example don't work or h3 for example in my app.

It shows up correctly in the editor

**Comment by jeffsee55** [comment]
Can you provide a reproduction or at least a screenshot of what you're seeing?

**Reply by jumptrnr** [reply]
For sure - inside editor: 
<img width="589" alt="image" src="https://user-images.githubusercontent.com/1178088/231820224-65f333df-1d61-47d0-8d24-338884f4464c.png">

As rendered on page: 
<img width="966" alt="image" src="https://user-images.githubusercontent.com/1178088/231820321-a5b8ab83-b295-4b09-a8d7-91489d0c9e16.png">

**Reply by jeffsee55** [reply, marked as answer]
It'll depend on how your css is set up, the `TinaMarkdown` component just renders html tags without any styling, it's up to you to provide that, I'm guessing your site has a css reset of some sort that makes h1 tags and blockquotes unstyled. To be sure, can you show what the elements are by inspecting your page (eg. is the "H1 Text" really rendering an `h1` html tag)?

**Reply by jumptrnr** [reply]
I see - when I comment out the tailwind css header I start seeing the formatting.  (am not using tailwind on my site).  I'll dig around and see how to style each element that comes back.  Appreciate your responses!
